integration-tests/src/test/resources/sitconfig/configoverrides/jdbc-JdbcTestDataSource-0.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<jdbc:jdbc-data-source
+    xmlns:jdbc="http://xmlns.oracle.com/weblogic/jdbc-data-source"
+    xmlns:jdbcf="http://xmlns.oracle.com/weblogic/jdbc-data-source-fragment"
+    xmlns:s="http://xmlns.oracle.com/weblogic/situational-config" >
+    <jdbc:name>JdbcTestDataSource-0</jdbc:name>
+    <jdbc:jdbc-connection-pool-params>
+        <jdbc:initial-capacity jdbcf:combine-mode="replace">2</jdbc:initial-capacity>
+        <jdbc:max-capacity jdbcf:combine-mode="replace">12</jdbc:max-capacity>
+        <jdbc:test-connections-on-reserve jdbcf:combine-mode="add">true</jdbc:test-connections-on-reserve>
+        <jdbc:connection-harvest-max-count jdbcf:combine-mode="add">7</jdbc:connection-harvest-max-count>
+        <jdbc:inactive-connection-timeout-seconds jdbcf:combine-mode="add">120</jdbc:inactive-connection-timeout-seconds>
+    </jdbc:jdbc-connection-pool-params>
+    <jdbc:jdbc-driver-params>
+        <jdbc:url jdbcf:combine-mode="replace">jdbc:oracle:thin:@//slc11smq.us.oracle.com:1583/w18ys12c.us.oracle.com</jdbc:url>
```

Comment:
We can not use the DB instance inside Oracle Network e.g.@slc11smq.us.oracle.com:1583/w18ys12c.us.oracle.com
Need to use create a lightweight mySQL db instance inside same K8s Cluster.  We can create utility to create a mySQL instance using mysql-operator opensource project.

Reply:
Will replace it with mysql db

Reply:
SECURITY VIOLATION
do not check in the name of oracle internal servers into public repositories

Reply:
replaced oracle with mysql db and removed all internal server references